ivy/functional/ivy/creation.py

```diff
@@ -1691,13 +1691,53 @@ def full(
     )
 
 
+@handle_exceptions
 @handle_backend_invalid
 @handle_nestable
-@handle_array_like_without_promotion
```

Comment:
not sure why we've removed this decorator

Reply:
for `to_dlpack`, we can add these no problem as paramter is array.